pwa-devdocs/src/tutorials/intercept-a-target/modify-talon-results/index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Modify talon results
+---
+
+In core Magento development:
+
+> A plugin, or interceptor, is a class that modifies the behavior of public class functions by intercepting a function call and running code before, after, or around that function call.
+> This allows you to substitute or extend the behavior of original, public methods for any class or interface.
+
+See: [Plugins (Interceptors)][]
+
+PWA Studio's extensibility framework provides a similar feature that allows you to intercept a talon call and surround it with custom logic.
+This is useful if you want to add tracking logic or alter the incoming or outgoing values for a talon.
+
+This tutorial teaches you how to create an extension that intercepts a talon and changes the results.
+
+## Tasks overview
+
+1. Initialize the project
+2. Create and register an intercept file
+3. Define the intercept file
+4. Make a data fetch hook
+5. Create the talon wrapper module
+6. Update the package entry point
+7. Test on a local instance
```

Comment:
Not for nothing but this might be easier to follow for someone new to all of this if the order was 

1. Initialize the project
2. Make a data fetch hook
3. Create the talon wrapper module
4. Define the intercept file
5. Create and register an intercept file
6. Update the package entry point
7. Test on a local instance

This way you're writing your extension code first (steps 1-3) and then connecting it to PWA Studio (steps 4-6).

Reply:
That's a fair point, but I wrote it this way because the focus of the tutorial is to connect code to PWA Studio, so I prioritized those steps higher because they will be the same for most developers.